Cargo.toml

```diff
@@ -88,7 +88,7 @@ gethostname = "0.5"
 git-version = "0.3"
 group = "0.13"
 hex = { version = "0.4", features = ["serde"] }
-hickory-resolver = { version = "0.24", default-features = false, features = ["system-config"] }
+hickory-resolver = { version = "0.25.0-alpha", default-features = false, features = ["system-config", "tokio-runtime"] }
```

Comment:
Fortunately, `hickory-resolver` is under maintenance. Unfortunately, `0.25` has not landed yet.